options.ts

```diff
@@ -1222,6 +1222,11 @@ const options = mkOptions(OPTIONS, {
     },
 
     dummy: opt(true),
+    hyprland: {
+        gaps: opt(2.4),
+        inactiveBorder: opt('#282828'),
+        gapsWhenOnly: opt(false),
+    },
```

Comment:
Was this added by mistake?

Reply:
oh yeah that was a mistake. Should have been on another branch